oligo_designer_toolsuite/oligo_specificity_filter/_filter_blastn.py

```diff
@@ -60,18 +76,28 @@ def __init__(
             "query_start",
             "query_end",
             "query_length",
-        ],
-    ):
+            "query_sequence",
+            "reference_start",
+            "reference_end",
+            "reference_sequence",
+            "reference_strand"
+        ], #TODO: update based on ai filters
```

Comment:
Do not add the additional fields for AI filters here per default. This would mean that we always read in / process more information than needed by default which is not good. This should move to AI filters definition and be updated then.